environment.yml

```diff
@@ -20,6 +20,7 @@ dependencies:
   - elasticsearch <7.14
   - elasticsearch-dsl
   - opensearch-py
+  - fabric
```

Comment:
invoke and paramiko should be added here as well if they're used directly. Also they should be added the the setup.cfg.